src/components/layout/HeaderMenu.vue

```diff
@@ -78,6 +87,9 @@ q-tabs(
           :class="[el.filter === localFileter ? 'active-tab': '']"
         )
     div.custom-separator
+  ResolveMenu(
+    v-if="isTestnet"
+  )
 </template>
 
 <style lang="sass">
```

Comment:
We do expect your code to have the same criteria across files and the style language would be one of them. I've seen you use `sass` sometimes and `scss` some others. Seems that's the case and is almost the same effort to go all for one option or the other, I will suggest to use `scss` because all `css` code is still `scss`  so you can have both of them in the same block and `sass` is very restrictive.

```suggestion
<style lang="scss">
```


**suggestion**: don't do this by hand. There are lots of online tools to translate from `sass` to `scss`. You just have to copy and paste, press convert and copy-paste back.